src/guiguts/application.py

```diff
@@ -190,7 +190,7 @@ def quit_program(self, *args: Any) -> None:
 
     def help_about(self, *args: Any) -> None:
         """Display a 'Help About' dialog."""
-        help = """Guiguts - an application to support creation of ebooks for PG
+        help_message = """Guiguts - an application to support creation of ebooks for PG
```

Comment:
I realize this was already this way, so this is as much a question for @windymilla as anything: Should PG be spelled out. _**We**_ all know what PG is, but "Project Gutenberg" sounds a bit more formal.

Instead of `help_message`, the variable could be "help_about" since that's how it's described in the comment, and more specific.

Reply:
Probably would be better to say Project Gutenberg, but maybe not in this PR. When making a "Help About" dialog, I just took that line from the first line of GG2's [README](https://github.com/DistributedProofreaders/guiguts-py/blob/master/README.md), so at some point we should probably revisit what is written in both places.